overlord/snapstate/handlers.go

```diff
@@ -1902,6 +1916,19 @@ func (m *SnapManager) undoLinkSnap(t *state.Task, _ *tomb.Tomb) error {
 	return nil
 }
 
+func countExcludedRevs(items []snap.Revision, seq []*snap.SideInfo) int {
+	var found int
+	for _, rev := range items {
+		for _, si := range seq {
+			if si.Revision == rev {
+				found++
+			}
+		}
+	}
+
+	return len(items) - found
```

Comment:
It would be nice to have a short docstring on this function (explaining what the two parameters contain), because here I was expecting just a `return found`... :-)

Reply:
Added and renamed the parameters as well. Thanks!